common/src/main/java/org/wso2/testgrid/common/config/DeploymentConfig.java

```diff
@@ -0,0 +1,107 @@
+/*
```

Comment:
All the public methods must include java doc comments. IMO this should be applicable for private methods as well considering future developers who will take over